qcommon/src/org/qcmg/common/model/ProfileType.java

```diff
@@ -101,7 +103,7 @@ public static ProfileType getType2(String s) {
 	    	ext = s.substring(i+1).toLowerCase();
 	    }
 	    
-	    if (BAM_EXTENSION.equals(ext) || SAM_EXTENSION.equals(ext))
+	    if (BAM_EXTENSION.equals(ext) || SAM_EXTENSION.equals(ext) || ext.equals(CRAM_EXTENSION))
```

Comment:
if ext is null here, then a NPE will be thrown, whereas if you do `CRAM_EXTENSION.equals(ext)` then it will never throw a NPE because `CRAM_EXTENSION` is a non-null constant

Reply:
I agree and fixed